.github/workflows/macOS.yml

```diff
@@ -16,23 +17,39 @@ jobs:
     steps:
       - uses: actions/checkout@v1
 
-      - name: Install dependencies from hombrew
+      - name: Install dependencies from homebrew
         run: |
           brew update
-          brew upgrade
           brew bundle --file=./.github/workflows/Brewfile
           brew link qt -f
 
-      - name: Build custom Python as a framework
+      - name: Build OpenSSL
+        run: |
+          wget https://www.openssl.org/source/openssl-${OPENSSL_VER}.tar.gz
```

Comment:
Should we have checksum verification for this, like for the windows builds in ykman-qt?

Reply:
Yes, it's a good addition for all tarballs we download in this workflow, but let's do it in separate PRs.